File: rides-service/src/main/java/io/simakkoi9/ridesservice/config/LoggingConfig.java
package io.simakkoi9.ridesservice.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.filter.CommonsRequestLoggingFilter;

@Configuration
public class LoggingConfig {
    @Bean
    public CommonsRequestLoggingFilter logFilter() {
        CommonsRequestLoggingFilter filter = new CommonsRequestLoggingFilter();

        filter.setIncludeQueryString(true);
        filter.setIncludePayload(false);
        filter.setIncludeHeaders(false);
        filter.setIncludeClientInfo(true);

        filter.setAfterMessagePrefix("Incoming Request: ");
        return filter;
    }
}

File: driver-service/src/main/kotlin/io/simakkoi9/driverservice/config/LoggingConfig.kt
package io.simakkoi9.driverservice.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.web.filter.CommonsRequestLoggingFilter

@Configuration
class LoggingConfig {
    @Bean
    fun logFilter(): CommonsRequestLoggingFilter {
        val filter = CommonsRequestLoggingFilter()
        filter.setIncludeQueryString(true)
        filter.setIncludePayload(false)
        filter.setIncludeHeaders(false)
        filter.setIncludeClientInfo(true)
        filter.setAfterMessagePrefix("Incoming Request: ")
        return filter
    }
}